services/api/src/typeDefs.js

```diff
@@ -897,6 +897,15 @@ const typeDefs = gql`
     openshiftProjectPattern: String @deprecated(reason: "No longer in use")
     kubernetes: Kubernetes
     kubernetesNamespacePattern: String @deprecated(reason: "No longer in use")
+    """
+    Pending changes tell us if we need to redeploy an environment
+    """
+    pendingChanges: [EnvironmentPendingChanges]
+  }
+
+  type EnvironmentPendingChanges {
+    type: String
+    details: String
```

Comment:
Also expose the change date?
```suggestion
  type EnvironmentPendingChanges {
    type: String
    details: String
    date: String
```